### ISSUE TITLE
Doesn't work in IE

### ISSUE THREAD

**Issue by prettytamben** [idiotWu/smooth-scrollbar#33]
Doesn't seem to work in IE and Edge. (windows 7 & windows 10) smooth-scrollbar v7.1.1

Demo page:
   HTML1300: Navigation occurred.
   File: smooth-scrollbar
   HTML1509: Unmatched end tag.
   File: smooth-scrollbar, Line: 67, Column: 110
   HTML1509: Unmatched end tag.
   File: smooth-scrollbar, Line: 68, Column: 114
   SCRIPT438: Object doesn't support property or method 'assign'
   File: app.js, Line: 2, Column: 12550

My tests: 
   HTML1300: Navigation occurred.
   File: index.html
   SCRIPT438: Object doesn't support property or method 'assign'
   File: smooth-scrollbar.js, Line: 2, Column: 6724

**Comment by idiotWu** [comment]
That's seemed to be a problem with babel [config](https://github.com/idiotWu/smooth-scrollbar/blob/develop/.babelrc#L6).

Can anyone tell me what's the meaning of each option in [babel runtime transformer](https://babeljs.io/docs/plugins/transform-runtime/#usage)?

**Comment by prettytamben** [comment]
I see the demo works now, thanks! However the 'dist' folder js version doesn't.

**Comment by idiotWu** [comment]
Oh babel config is still broken :(

**Comment by idiotWu** [comment]
@prettytamben I found no workaround the conflict caused by `babel-transform-runtime-plugin`, so I'm going to remove all `Object.assign` in my code 😢

**Comment by prettytamben** [comment]
What are the consequences?

**Comment by idiotWu** [comment]
@prettytamben `Object.assign` requires a polyfill in IE, but if I enable `polyfill` option in runtime transformer, it will generate code like:

```javascript
import _Object$defineProperty from 'babel-runtime/core-js/object/define-property';
import _Object$keys from 'babel-runtime/core-js/object/keys';
Object.defineProperty(exports, "__esModule", {
  value: true
});
```

That is a well-known bug in babel@6: https://phabricator.babeljs.io/T2877, and hasn't been resolved yet 😢

**Comment by idiotWu** [comment]
Try update to version 7.1.3

**Comment by prettytamben** [comment]
Thanks, it works!

**Comment by idiotWu** [comment]
@prettytamben actually it's still failing in IE10... I am looking for another workaround.

**Comment by idiotWu** [comment]
Should be resolved in version 7.1.4 👏 🎉

**Comment by prettytamben** [comment]
Genius! :)
